OpenEXR/IlmImf/ImfFastHuf.cpp

```diff
@@ -667,7 +667,7 @@ FastHufDecoder::decode
             }
 
             Int64 id = _ljOffset[codeLen] + (buffer >> (64 - codeLen));
-            if (id < _numSymbols)
+            if (id < Int64(_numSymbols))
```

Comment:
static_cast<Int64>

Reply:
I've converted all constructor casts to static_cast